src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/DeleteResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,37 @@
+# This is a rough draft feature for future use.
+@ignore
+Feature: Resources "Delete" Operation validations
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+              And request made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave",
+                        "effectiveBeginDate": "2024-05-14",
+                        "effectiveEndDate": "2024-05-14",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 201
+
+        @ignore
+        Scenario: Verify deleting a specific resource by ID
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 204
+
+        @ignore
+        Scenario: Verify error handling when deleting a non existing resource
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/00112233445566"
+             Then it should respond with 400
```

Comment:
Should be 404 when the URL does not exist.

Reply:
I added 2 scenarios for deleting: 
1. First Scenario will check a deleted resource. It should return a 404, and this is the response body
```
                    {
                        "detail": "Resource to delete was not found.",
                        "type": "urn:ed-fi:api:not-found",
                        "title": "Not Found",
                        "status": 404,
                        "correlationId": null
                    }
```
2. Second Scenario will check a non valid ID. Currently it returns a 400 with the following body 
```
                 {
                        "detail": "Data validation failed. See 'validationErrors' for details.",
                        "type": "urn:ed-fi:api:bad-request:data",
                        "title": "Data Validation Failed",
                        "status": 400,
                        "correlationId": null,
                        "validationErrors": {
                            "$.id": [
                                "The value 'XXXXXXXXXXX' is not valid."
                            ]
                        }
                    }
```

Reply:
Good catch: a realistic (but does not exist) identifier, and something that is completely fake.